packages/vitest/src/node/cli-api.ts

```diff
@@ -92,7 +97,7 @@ export async function startVitest(
     return ctx
   }
 
-  if (ctx.config.watch)
+  if (ctx.config.watch || ctx.config.browser)
```

Comment:
Should not be needed

Reply:
Removed it and the server didn't even respond. So no change, let me know if you have a solution.

Reply:
This should be unified. From a code perspective we can't just add `|| ctx.browser` everywhere. Create an abstraction